docs/development/project_structure.md

```diff
@@ -0,0 +1,39 @@
+# Chapter 1: Understanding project structure
+
+The project is divided into two main parts:
+
+1. The vscode-ansible extension (client)
+2. The ansible-language-server (server)
+
+Each part has its own package.json files, tests, and tsconfigs. This modularity
+helps in code navigation and development.
+
+!!! tip
```

Comment:
```suggestion
[!TIP]
```
This is a [new markdown feature](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts) that might be cool to include, optional nitpick.

Reply:
I was following the syntax for [mkdocs admonitions](https://squidfunk.github.io/mkdocs-material/reference/admonitions/). But I can give this a try.

Reply:
I tested it out, and it does not render well in readthedocs. I will stick to the previous style.

Reply:
This is how it looks on the documentation website:
![Screenshot 2024-07-29 at 8 12 42 PM](https://github.com/user-attachments/assets/b210abc9-cb82-403e-95d5-ef7e12e38b96)

Reply:
Looks good!